demos/warnings/README.md

```diff
@@ -25,3 +25,23 @@ unclassified:
           return builder.buildOptional();
         example: "somefile.txt:2:SeriousWarnings:SomethingWentWrong"
 ```
+
+This `Example Parser` parser will parse the following warning from the console log:
+```
```

Comment:
there's a few errors from codacy around the markdown formatting:
https://app.codacy.com/manual/casz/configuration-as-code-plugin/pullRequest?prid=4328906

Reply:
Seems that a [remark-lint](https://github.com/remarkjs/remark-lint) config file is missing? How do you check that locally?

Reply:
because remark-lint relies on nodejs we don't enforce it and use codacy to catch any linting errors :)

Reply:
Feel free to send a PR :)